regnet/regnet_schema.json

```diff
@@ -77,7 +82,6 @@
     "base_properties": {
       "type": "object",
       "properties": {
-        "name": { "type": "string" },
```

Comment:
Oh I see, you changed the validation. I think that this is incorrect and should not be done this way. It begins to intermingle the idea of "metadata" and "model structure" which we have been separating with this model representation.

Reply:
Your original approach makes sense: I've changed both `petrinet_schema` and `regnet_schema` to keep `name` and `description` of RegNet edges and PetriNet transitions in `properties`.